pkg/collector/reader.go

```diff
@@ -104,8 +104,12 @@ func (c *Collector) resetCurrValue() {
 
 // resetBPFTables reset BPF module's tables
 func (c *Collector) resetBPFTables() {
-	c.modules.Table.DeleteAll()
-	c.modules.TimeTable.DeleteAll()
+	if c.modules != nil && c.modules.Table != nil {
```

Comment:
@rootfs , do you know where we init modules and tables?

Reply:
the tables are auto filled; the delete functions remove existing table entries, the ebpf program will fill the tables again.